applications/RomApplication/python_scripts/hrom_training_utility.py

```diff
@@ -172,9 +201,11 @@ def __GetHRomTrainingDefaultSettings(cls):
             "projection_strategy": "galerkin",
             "include_conditions_model_parts_list": [],
             "include_elements_model_parts_list": [],
+            "initial_candidate_elements_model_part_list" : [],
+            "initial_candidate_conditions_model_part_list" : [],
             "include_nodal_neighbouring_elements_model_parts_list":[],
             "include_minimum_condition": false,
-            "include_condition_parents": true
+            "include_condition_parents": false
```

Comment:
In the worst case, it won't use the info of the parent element. Still, it is generally more likely to fail because we need an associated parent element (e.g. Convection-Diffusion, Fluid-Dynamics, CompressiblePotenital).

Reply:
![image](https://github.com/KratosMultiphysics/Kratos/assets/61457043/d0e0977e-6231-4e83-aa23-fe54a8844025)